### PR TITLE
DEVPROD-13256 AttributeError: Unknown attribute group_id

### PR DESCRIPTION
**Pull request by namazgurbanovmdb**

### CODE REVIEW

File: src/evergreen/tst.py [hunk 1]
```diff
@@ -13,12 +13,10 @@
 class Logs(_BaseEvergreenObject):
     """Representation of test logs from evergreen."""
 
-    url = evg_attrib("url")
     line_num = evg_attrib("line_num")
-    url_raw = evg_attrib("url_raw")
-    url_lobster = evg_attrib("url_lobster")
+    url = evg_attrib("url")
     url_parsley = evg_attrib("url_parsley")
-    log_id = evg_attrib("log_id")
```
Comment (namazgurbanovmdb) [inline src/evergreen/tst.py:21]: Removed log_id because it does not exist in evergreen test log object

File: src/evergreen/tst.py [hunk 2]
```diff
@@ -13,12 +13,10 @@
 class Logs(_BaseEvergreenObject):
     """Representation of test logs from evergreen."""
 
-    url = evg_attrib("url")
     line_num = evg_attrib("line_num")
-    url_raw = evg_attrib("url_raw")
-    url_lobster = evg_attrib("url_lobster")
```
Comment (namazgurbanovmdb) [inline src/evergreen/tst.py:19]: Removed url_lobster because it does not exist in evergreen test log object. lobster has been deprecated.

**Review by rongodb — APPROVED** [review]
LGTM

File: src/evergreen/tst.py [hunk 3]
```diff
@@ -36,13 +34,13 @@ def stream(self) -> Iterable[str]:
 class Tst(_BaseEvergreenObject):
     """Representation of a test object from evergreen."""
 
-    task_id = evg_attrib("task_id")
+    end_time = evg_datetime_attrib("end_time")
+    group_id = evg_datetime_attrib("group_id")
```
Comment (namazgurbanovmdb) [inline src/evergreen/tst.py:38]: added group_id to test object because it is being used in build baron

File: src/evergreen/tst.py [hunk 4]
```diff
@@ -13,12 +13,10 @@
 class Logs(_BaseEvergreenObject):
     """Representation of test logs from evergreen."""
 
-    url = evg_attrib("url")
     line_num = evg_attrib("line_num")
-    url_raw = evg_attrib("url_raw")
-    url_lobster = evg_attrib("url_lobster")
+    url = evg_attrib("url")
```
Comment (namazgurbanovmdb) [inline src/evergreen/tst.py:17]: both test and test log class fields have been reorganized to match the order in evergreen documentation [here](https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tests/paths/~1tasks~1%7Btask_id%7D~1tests/get).

File: src/evergreen/tst.py [hunk 5]
```diff
@@ -36,13 +34,13 @@ def stream(self) -> Iterable[str]:
...
 
-    task_id = evg_attrib("task_id")
+    end_time = evg_datetime_attrib("end_time")
+    group_id = evg_datetime_attrib("group_id")
+    start_time = evg_datetime_attrib("start_time")
     status = evg_attrib("status")
-    test_id = evg_attrib("test_id")
+    task_id = evg_attrib("task_id")
     test_file = evg_attrib("test_file")
-    exit_code = evg_attrib("exit_code")
```
Comment (namazgurbanovmdb) [inline src/evergreen/tst.py:43]: Removed exit because it does not exist in evergreen test object

**Pull request merged**